### DISCUSSION TITLE
Not able to update to  1.1.21 on Linux Mint

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5905] [Issues] [answered]
Hi all,
I see an error when trying to update about empty folder:
pm ERR! ENOTEMPTY: directory not empty
What does it mean? how can it be fixed? Is there an option to update to older versions like 1.1.20?
Thanks.

**Comment by si458** [comment]
you need to be a little more descriptive...
where are you seeing the error?
does it say what folder?

to way to install a certain version is
1. stop meshcentral
2. remove node_modules comepletely
3. `npm install meshcentral@VERSIONNUM` - `meshcentral@1.1.20`
4. make sure you have `selfUpdate: false` set inside your config.json
5. start meshcentral

**Reply by ghost** [reply]
I see this error when I run serverupdate and in the terminal I enter _sudo systemctl status meshcentral_ in the terminal
This is what I see in the terminal

npm ERR! path /home/quizhead/meshcentral/node_modules/meshcentral
npm ERR! dest /home/quizhead/meshcentral/node_modules/.meshcentral-hCTqv3Eb
npm ERR! errno -39
npm ERR! ENOTEMPTY: directory not empty, rename '/home/quizhead/meshcentral/node_modules/meshcentral' -> '/home/quizhead/meshcentral/node_module>

**Reply by si458** [reply]
i think your server has got itself confused...
try the steps above i mentioned (REMEMEBER BACKUP EVERYTHING FIRST!)

**Comment by ghost** [comment, marked as answer]
Ok so removing the Node_Modules folder and reinstallation solved it and the server is not updated to 1.1.21
One question: Until now I didn't enter the SelfUpdate: false in the Config file so without it ,it means that the server is trying to update itself every day by default?

**Reply by si458** [reply]
the default for `selfUpdate` is false, so it wont attempt to self-update everyday after midnight,
unless you run the `npm install` OR `serverupdate` in the console OR run the web ui updater
HOWEVER i have just spotted the schema says its true or false BUT when checking the code,
you can actually specify a version you want to run and when you start meshcentral it will update to that version for you!
so i need to add this to the schema file
